examples/GTEx/README.md

```diff
@@ -0,0 +1,59 @@
+# Create a leakage-free dataset of tiles for GTEx
+
+`extract_tile_pw_gtex.py` is a Python script proposed as reference to retrieve a reproducible dataset of tiles using a collection of WSIs from the [GTEx](https://gtexportal.org/home/) public repository. In particular, it can be easily integrated in deep learning pipeline for computational pathology.
```

Comment:
should we use plural here? -> `deep learning pipeline(s)`

Reply:
ok